src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs

```diff
@@ -1566,10 +1575,7 @@ public void DisableSearchTextBox()
         /// </summary>
         internal void Close()
         {
-            TimedOut = false;   // reset the timedout screen 
-            InitialResultsLoaded = false;   // reset the loading screen settings
-            RequestShowFileDialog -= OnRequestShowFileDialog;
-            nonHostFilter.ForEach(f => f.PropertyChanged -= filter_PropertyChanged);
```

Comment:
I think youneed to unsubscribe these handlers somewhere, if you are going to use this method just as a reset of the view, then I think you need to a dispose call to this viewModel higher up, like at DynamoView shutdown or something like that.

Reply:
OK, got it.

Reply:
Yup, done now! Thanks for the review, @mjkkirschner